components/script/dom/htmloptionelement.rs

```diff
@@ -72,6 +75,30 @@ impl HTMLOptionElement {
         )
     }
 
+    // https://html.spec.whatwg.org/multipage/#dom-option
+    pub fn Option(
+        window: &Window,
+        text: DOMString,
+        value: Option<DOMString>,
+        default_selected: bool,
+        selected: bool,
+    ) -> Fallible<DomRoot<HTMLOptionElement>> {
+        let document = window.Document();
+        let option = HTMLOptionElement::new(local_name!("option"), None, &document);
```

Comment:
We should be using the `create_html_element` function from components/script/dom/create.rs instead, according to the specification.

Reply:
Looks like Image and Audio constructors should be updated to use `create_html_element` as well:

https://github.com/servo/servo/blob/1e0f2e59b7fe5238e42b53099cc992ca6d2ec4d3/components/script/dom/htmlimageelement.rs#L1254-L1260

https://github.com/servo/servo/blob/1e0f2e59b7fe5238e42b53099cc992ca6d2ec4d3/components/script/dom/htmlaudioelement.rs#L52-L54

https://html.spec.whatwg.org/multipage/embedded-content.html#dom-image
https://html.spec.whatwg.org/multipage/media.html#dom-audio

Reply:
Indeed!